libs/core/src/scss/_global-styles.scss

```diff
@@ -63,186 +32,10 @@ $modal-large-screen-min-height: $modal-header-height + $modal-expected-footer-he
   -webkit-tap-highlight-color: transparent;
 }
 
-@function get-drawer-additional-padding-top() {
-  $modal-close-button-inner-height: utils.size('l'); // Todo: Move to / get from shared var
-  $modal-close-button-vertical-margin: utils.size('xxxs') * 2; // Todo: Move to / get from shared var
-  $modal-header-vertical-padding: 3px * 2; // 3px defined by Ionic
-  $modal-header-total-height: $modal-close-button-inner-height +
-    $modal-close-button-vertical-margin + $modal-header-vertical-padding;
-  $modal-header-vertical-center: math.round($modal-header-total-height * 0.5);
-  @return $modal-header-vertical-center;
-}
-
 body.allow-background-scroll {
   overflow: initial !important;
 }
```

Comment:
Despite the ambiguous naming this is only used in modals, so it can be moved to the partial as well 😊